experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -517,6 +540,10 @@ func (r *ExpanderReconciler) evaluateAndSavePlan(ctx context.Context, logger log
 		err = fmt.Errorf("Evaluate Failed: %s", result.Error.Message)
 		return values, updated, "EvaluateStatusFailed", err
 	}
+	if expanderDebugLogEnabled {
+		logger.Info(expanderDebugLog(cr) + fmt.Sprintf("---sent expander request: %v, received results: %v", evaluateRequest, result))
+		r.Recorder.Event(cr, "Normal", fmt.Sprintf("Expander stage %s evaluation completed", expander.Name), expanderDebugLog(cr)+fmt.Sprintf("---request: %v, result: %v", evaluateRequest, result))
```

Comment:
Not sure if there is a limit on what can be sent to an event. Nevertheless lets try it.